### PR TITLE
feat(molecule/breadcrumb): removed deprecated dependency

### PR DESCRIPTION
**Pull request by juanmaguitar**
removed deprecated dependency as explained here → https://jira.scmspain.com/browse/SUIC-81

### CODE REVIEW

**Review by midudev — DISMISSED** [review]

File: components/molecule/breadcrumb/src/utils.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+$breakpoints: (
```
Comment (midudev) [inline components/molecule/breadcrumb/src/utils.scss]: Mmmm I think breakpoint-up is still valid in the next sui-theme: https://github.com/SUI-Components/sui-theme/blob/cf07c47c68541bdb8a2f550235e96fe52bd34d83/src/utils/_breakpoints.scss#L18

Why exactly do you need this utils file?

Comment (juanmaguitar) [reply components/molecule/breadcrumb/src/utils.scss]: I'm getting this error if I remove the utils file

Captura de pantalla 2019-06-03 a las 14 47 23

I've put in this file all functions needed for the current code so I can remove the dependency without errors

Comment (jcger) [reply components/molecule/breadcrumb/src/utils.scss]: you cannot use media-breakpoint-`down`, this task is about refactor in order to use media-breakpoint-`up` and make the component mobile first

Comment (juanmaguitar) [reply components/molecule/breadcrumb/src/utils.scss]: oks, fixed!

**Review by JoanClaret — APPROVED** [review]

**Review by midudev — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Pull request merged**